script-runner/stack-test

```diff
@@ -2,14 +2,23 @@
 
 # usage: ./stack-test test4.2
 
-# on Mac (and OS's with different `mktemp` versions)
-# usage: ALT_MKTEMP=1 ./stack-test test4.2
+# on Mac
+# usage: DARWIN=1 ./stack-test test4.2
 
 set -e
 set -x
 
+
+if [ -z $DARWIN ]; then
```

Comment:
Couldn't we use `$OSTYPE` or even the `uname` output for this?

Reply:
Yes, great idea. This was originally a small workaround for me to run (on Mac) the script which @cleverca22 wrote. Good to make it more formal.